docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,160 @@
+---
```

Comment:
This file will need to be a `.mdx` instead of a `.md`.

Reply:
Thanks for the feedback. This is what I expected, but I didn't manage to make this readable in my local dev environment.. Maybe I was missing a trick.
Anyways, I thought I write it like this and convert it in the end. Let me keep this comment open as a reminder